commune.lic

```diff
@@ -0,0 +1,367 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#commune
+=end
+
+custom_require.call(%w(common common-items common-arcana drinfomon))
+
+class Commune
+  include DRC
+  include DRCI
+  include DRCA
+
+  ######################## ITEM ACTIONS ########################
+
+  def get_cleric_item(item)
+    case bput("get #{item} from my #{@theurgy_supply_container}", 'You get', 'You are already holding', 'I could not find', 'What were you referring to')
+	when 'You get' || 'You are already holding'
+	  return true
+	else
+	  return false
+	end
+  end
+
+  def stow_cleric_item(item)
```

Comment:
Does `stow_cleric_item` need to return true or false? I don't see anywhere the return value is used.

If you are using the return value, I recommend naming this `stow_cleric_item?` to be clear that it is a predicate.

Reply:
Fixed. it does not need true or false.